docs/geedocs/docsrc/answer/geeServerConfigAndSecurity/postgreSqlAuthentication.rst

```diff
@@ -47,12 +47,12 @@ PostgreSQL authentication
       -  The ``mod_authn_dbd`` module provides the authentication
          back-end with a PostgreSQL server. For more information about
          the module, see the
-         `mod_authn_dbd <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.2%2Fmod%2Fmod_authn_dbd.html&sa=D&sntz=1&usg=AFrqEzcLzX90MM12j49RNuWEy_X7y6wTrQ>`__
+         `mod_authn_dbd <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.4%2Fmod%2Fmod_authn_dbd.html&sa=D&sntz=1&usg=AFrqEzcLzX90MM12j49RNuWEy_X7y6wTrQ>`__
          section in the Apache documentation.
       -  The ``mod_auth_digest`` module is used for Digest
          authentication. For more information about ``mod_auth_digest``,
          see the
-         `mod_auth_digest <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.2%2Fmod%2Fmod_auth_digest.html&sa=D&sntz=1&usg=AFrqEzfaZylqvLk4_vXVqc3jCj7EJESuhw>`__
+         `mod_auth_digest <http://www.google.com/url?q=http%3A%2F%2Fhttpd.apache.org%2Fdocs%2F2.4%2Fmod%2Fmod_auth_digest.html&sa=D&sntz=1&usg=AFrqEzfaZylqvLk4_vXVqc3jCj7EJESuhw>`__
```

Comment:
the url does not seem to work. Can we updated to: http://httpd.apache.org/docs/2.4/mod/mod_auth_digest.html ?

Reply:
yeah. I will change it. Thanks @tst-nfarah